librz/core/cmd/cmd_help.c

```diff
@@ -430,696 +336,764 @@ RZ_IPI void rz_core_clippy_print(RzCore *core, const char *msg) {
+		if (!buf) {
+			RZ_LOG_ERROR("core: Out of memory!");
+			return RZ_CMD_STATUS_ERROR;
 		}
-		break;
+		rz_base64_decode(buf, argv[i], -1);
+		rz_cons_println((const char *)buf);
+	}
+	free(buf);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_check_between_handler(RzCore *core, int argc, const char **argv) {
+	st64 f = rz_num_math(core->num, argv[1]);
+	st64 m = rz_num_math(core->num, argv[2]);
+	st64 l = rz_num_math(core->num, argv[3]);
+	core->num->value = RZ_BETWEEN(f, m, l);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_boundaries_prot_handler(RzCore *core, int argc, const char **argv) {
+	const char *mode = rz_str_trim_head_ro(argv[0]);
+	RzList *list = rz_core_get_boundaries_prot(core, -1, mode, "search");
+	if (!list) {
+		RZ_LOG_ERROR("core: Failed to get boundaries protection values in RzList");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	RzListIter *iter;
+	RzIOMap *map;
+	rz_list_foreach (list, iter, map) {
+		rz_cons_printf("0x%" PFMT64x " 0x%" PFMT64x "\n", map->itv.addr, rz_itv_end(map->itv));
+	}
+	rz_list_free(list);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_djb2_hash_handler(RzCore *core, int argc, const char **argv) {
+	for (int i = 1; i < argc; i++) {
+		ut32 hash = (ut32)rz_str_djb2_hash(argv[i]);
+		rz_cons_printf("0x%08x\n", hash);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_flush_console_handler(RzCore *core, int argc, const char **argv) {
+	rz_cons_flush();
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_bitstring_handler(RzCore *core, int argc, const char **argv) {
+	ut64 n = 0;
+	n = rz_num_get(core->num, argv[1]);
+	char out[128] = RZ_EMPTY;
+	rz_str_bits(out, (const ut8 *)&n, sizeof(n) * 8, argv[2]);
+	rz_cons_println(out);
+
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_eval_expr_print_octal_handler(RzCore *core, int argc, const char **argv) {
+	ut64 n = rz_num_math(core->num, argv[1]);
+	rz_cons_printf("0%" PFMT64o "\n", n);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_init_time_values_handler(RzCore *core, int argc, const char **argv) {
+	rz_cons_printf("plug.init = %" PFMT64d "\n"
+		       "plug.load = %" PFMT64d "\n"
+		       "file.load = %" PFMT64d "\n",
+		core->times->loadlibs_init_time,
+		core->times->loadlibs_time,
+		core->times->file_open_time);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_num_to_units_handler(RzCore *core, int argc, const char **argv) {
+	char unit[8];
+	ut64 n = rz_num_math(core->num, argv[1]);
+	rz_num_units(unit, sizeof(unit), n);
+	rz_cons_println(unit);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_set_last_eval_expr_handler(RzCore *core, int argc, const char **argv) {
+	if (core->num->dbz) {
+		RZ_LOG_ERROR("core: RzNum ERROR: Division by Zero\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	rz_num_math(core->num, argv[1]);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_show_value_handler(RzCore *core, int argc, const char **argv) {
+	ut64 n;
+	if (argc == 1) {
```

Comment:
You can use `ut64 n = argc == 1 ? core->num->value : rz_num_math()`

Reply:
Though I would use `argc < 2` to show the logic better instead of direct comparison.